modules/transactions/src/test/java/org/apache/ignite/internal/tx/storage/state/rocksdb/RocksDbTxStateStorageTest.java

```diff
@@ -56,18 +59,25 @@ protected TxStateRocksDbTableStorage createTableStorage() {
         return new TxStateRocksDbTableStorage(
                 TABLE_ID,
                 3,
-                workDir,
-                scheduledExecutor,
-                executor,
-                () -> 1_000
+                sharedStorage
         );
     }
 
+    @Override
+    @BeforeEach
+    protected void beforeTest() {
+        sharedStorage = new TxStateRocksDbSharedStorage(workDir, scheduledExecutor, executor, () -> 0);
+        sharedStorage.start();
+
+        super.beforeTest();
+    }
+
     @Override
     @AfterEach
     protected void afterTest() {
         super.afterTest();
 
+        sharedStorage.close();
         IgniteUtils.shutdownAndAwaitTermination(scheduledExecutor, 10, TimeUnit.SECONDS);
         IgniteUtils.shutdownAndAwaitTermination(executor, 10, TimeUnit.SECONDS);
```

Comment:
I would make all calls in this method in IgniteUtils#closeAll, if desired.

Reply:
ok